<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WReservation</class>
 <widget class="QWidget" name="WReservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>607</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnCopyLast">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>222</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Copy last reservation</string>
        </property>
        <property name="text">
         <string>Copy last</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>222</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Save</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCheckIn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>222</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Checkin</string>
        </property>
        <property name="text">
         <string>Checkin</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/checkin.png</normaloff>:/images/checkin.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPrint">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>222</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Print</string>
        </property>
        <property name="text">
         <string>Print</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/printer.png</normaloff>:/images/printer.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSendConfirmation">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>222</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Print</string>
        </property>
        <property name="text">
         <string>Email confirmation</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/email.png</normaloff>:/images/email.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancelation">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>222</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Cancel reservation</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/garbage.png</normaloff>:/images/garbage.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRecheckin">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>Re-checkin</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/goto.png</normaloff>:/images/goto.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRevive">
        <property name="maximumSize">
         <size>
          <width>2222</width>
          <height>2222</height>
         </size>
        </property>
        <property name="text">
         <string>Revive</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/heart-pulses.png</normaloff>:/images/heart-pulses.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbGroup">
        <property name="text">
         <string>Group #</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EQLineEdit" name="leGroup">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetGroupParams">
        <property name="maximumSize">
         <size>
          <width>222</width>
          <height>222</height>
         </size>
        </property>
        <property name="text">
         <string>Group params</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddRemoveGroup">
        <property name="maximumSize">
         <size>
          <width>222</width>
          <height>222</height>
         </size>
        </property>
        <property name="text">
         <string>Add/Remove from group</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancelGroup">
        <property name="maximumSize">
         <size>
          <width>222</width>
          <height>222</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel Group</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/garbage.png</normaloff>:/images/garbage.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>531</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnTracking">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/tracking.png</normaloff>:/images/tracking.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnHelp">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tab">
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="tabsClosable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lbAuthor">
        <property name="text">
         <string>Author:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>646</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbLastModify">
        <property name="text">
         <string>Last modify:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
